src/pytest_redis/factories.py

```diff
@@ -200,6 +208,8 @@ def redisdb_factory(request):
 
         redis_host = proc_fixture.host
         redis_port = proc_fixture.port
+        redis_username = proc_fixture.username if hasattr(proc_fixture, "username") else None
+        redis_password = proc_fixture.password if hasattr(proc_fixture, "password") else None
```

Comment:
at this moment, all proc_fixtures have password attribute (and username, but regular executor does not actually needs it)

Reply:
I can either leave the username attribute in regular executor and remove the hasattr checks, or remove the username attribute in regular executor and leave the hasattr at least for the username